src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -30,10 +31,12 @@
 	actions={route !== 'settings'}
 />
 
-<main class="pt-12">
-	<svelte:component this={cmpLoaders}>
-		<slot />
-	</svelte:component>
-</main>
+<div use:pointerEventsHandler>
```

Comment:
Given that we want to hijack the tokens list only, I don't think this div should find place in the root layout.

Reply:
I will move it back to the token list. I had moved it there to include the margins around the tokens list, referring to this image:

<img width="1536" alt="Capture d’écran 2024-07-18 à 06 43 00" src="https://github.com/user-attachments/assets/7de7359d-6816-4f73-96ef-983ec431f1ce">